dask/base.py

```diff
@@ -571,15 +571,15 @@ def compute(*args, **kwargs):
 
 def visualize(*args, **kwargs):
     """
-    Visualize several dask graphs at once.
+    Visualize a dask low level graph.
 
     Requires ``graphviz`` to be installed. All options that are not the dask
     graph(s) should be passed as keyword arguments.
 
     Parameters
     ----------
     dsk : dict(s) or collection(s)
```

Comment:
This parameter is, however, wrong. `*args` is the set of dask graphs to show, one or more of them.